torch/nn/quantized/modules/conv.py

```diff
@@ -244,17 +272,7 @@ def from_float(cls, mod):
             mod (Module): a float module, either produced by torch.quantization
               utilities or provided by the user
         """
-        assert type(mod) == cls._FLOAT_MODULE, \
-            ' nnq.' + cls.__name__ + '.from_float only works for ' + \
-            cls._FLOAT_MODULE.__name__
-        assert hasattr(mod, 'qconfig'), \
-            'Input float module must have qconfig defined.'
-        if type(mod) == nni.ConvReLU1d:
-            activation_post_process = mod[1].activation_post_process
-            mod = mod[0]
-        else:
-            activation_post_process = mod.activation_post_process
-        return cls.get_qconv(mod, activation_post_process)
+        return from_float_helper(cls, mod)
```

Comment:
optional: thoughts on if instead of `from_float_helper` this logic could live in `torch.nn.quantized._ConvNd.from_float`, and then `torch.nn.quantized.Conv3d.from_float` could override it?  Seems like that is what's actually happening, but it's not obvious from the current names

Reply:
oh sure. will move that function to torch.nn.quantized._ConvNd.from_float